### PR TITLE
Include agent and workload statuses in the unit details.

### PR DESCRIPTION
**Pull request by frankban**
Fixes https://github.com/juju/juju-gui/issues/2668

### CODE REVIEW

**Comment by frankban** [comment]
QA:
- for deployed units, check that the agent and workload statuses are displayed in the detail view in the inspector;
- for uncommitted units, check that only the uncommitted status is included in that view.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7144/

File: jujugui/static/gui/src/app/components/inspector/unit-details/unit-details.js [hunk 1]
```diff
@@ -77,7 +62,7 @@ YUI.add('unit-details', function() {
       @param {Boolean} clickabl Whether the addresses are clickable.
       @returns {String} HTML of list.
     */
-    _getAddressList: function(address, portRanges, clickable) {
+    _generateAddresses: function(address, portRanges, clickable) {
```
Comment (mitechie) [inline jujugui/static/gui/src/app/components/inspector/unit-details/unit-details.js]: Is the addresses stuff all just drive by?

Comment (frankban) [inline jujugui/static/gui/src/app/components/inspector/unit-details/unit-details.js]: I just renamed the method for consistency, yes.

**Comment by hatched** [comment]
QA Notes:
- When viewing a fully deployed and operational PostgreSQL the unit status shows "Live Master (9.5.6)" but when I click through to the unit details it shows "Started - Live Master (9.5.6)". Other units which do not have a started status do not show anything. Should we show the 'Started' status for all without a workload status? Possible follow-up? 🤷‍♂️
- Uncommitted do not show any status in the unit list.  ✓
- Deploying Kubernetes core bundle...
- Pending statuses (waiting for machine) properly update and display. ✓
QA 🆗

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**